bazel/emscripten_toolchain/wasm_cc_binary.bzl

```diff
@@ -32,12 +32,13 @@ def _wasm_transition_impl(settings, attr):
 
     return {
         "//command_line_option:compiler": "emscripten",
-        "//command_line_option:crosstool_top": "@emsdk//emscripten_toolchain:everything",
         "//command_line_option:cpu": "wasm",
         "//command_line_option:features": features,
         "//command_line_option:dynamic_mode": "off",
         "//command_line_option:linkopt": linkopts,
         "//command_line_option:platforms": [platform],
+        # This is hardcoded to an empty cc_library because dlmalloc.bc
```

Comment:
The `.bc` extension is no longer used by emscripten.  Perhaps just call say "... because the malloc library is implicitly added..." ?

Reply:
Makes sense to me.

```suggestion
        # This is hardcoded to an empty cc_library because the malloc library
```